src/components/ui.css.ts

```diff
@@ -145,15 +167,17 @@ export const marginY = styleVariants(theme.space, (margin) => ({
   marginTop: margin,
   marginBottom: margin,
 }))
+
 export const gutter = styleVariants(theme.space, (val) => ({
   marginLeft: calc.multiply(val, -1),
   marginRight: calc.multiply(val, -1),
 }))
+
 export const radii = styleVariants(theme.radii, (borderRadius) => ({
   overflow: "hidden",
   borderRadius,
 }))
-export const order = styleVariants([0, 1, 2, 3], (order) => ({
+export const order = styleVariants({ 0: 0, 1: 1, 2: 2, 3: 3 }, (order) => ({
```

Comment:
Is this not typed to accept array objects in vanilla extract?

Reply:
Yeah, `styleVariants` is typed to accept a `Record` as it's first argument